test/common/config/subscription_factory_impl_test.cc

```diff
@@ -220,18 +220,9 @@ TEST_F(SubscriptionFactoryTest, FilesystemCollectionSubscription) {
   collectionSubscriptionFromUrl(fmt::format("file:///{}", file_path), {})->start({});
 }
 
-TEST_F(SubscriptionFactoryTest, FilesystemCollectionSubscriptionNonExistentFile) {
```

Comment:
How is this test relevant?

Reply:
@htuch  Really Sorry, My mistake :( . reverted back it and deleted  Test Cae "LegacySubscription"